cmd/telliot/main.go

```diff
@@ -31,75 +31,83 @@ import (
 )
 
 var ctx context.Context
-var cont tellorCommon.Contract
-var acc tellorCommon.Account
-var clt rpc.ETHClient
 
-func ExitOnError(err error, operation string) {
-	if err != nil {
-		fmt.Fprintf(os.Stderr, "%s failed: %s\n", operation, err.Error())
-		cli.Exit(-1)
-	}
-}
+// var cont tellorCommon.Contract
+// var acc tellorCommon.Account
+// var clt rpc.ETHClient
+
+// func ExitOnError(err error, operation string) {
+// 	if err != nil {
+// 		fmt.Fprintf(os.Stderr, "%s failed: %s\n", operation, err.Error())
+// 		cli.Exit(-1)
+// 	}
+// }
+
+func setup() (rpc.ETHClient, tellorCommon.Contract, tellorCommon.Account, error) {
```

Comment:
usually we do 
```suggestion
func setup() (rpc.ETHClient, *tellorCommon.Contract, *tellorCommon.Account, error) {
```
so that when there is an error can just 
```
return nil, nil, nil, errors.Wrap(err, "create rpc client instance")